src/components/pages/SignIn/index.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControlLabel,
+  Link,
+  Paper,
+  TextField,
+  Typography,
+} from "@mui/material";
+import { styled, useTheme } from "@mui/material/styles";
+import { useState } from "react";
+
+const StyledTextField = styled(TextField)({
+  marginTop: "25px",
+  width: "85%",
+  "& .MuiOutlinedInput-root": {
+    borderRadius: "40px",
+    height: 40,
+  },
+  "& .MuiInputBase-input": {
+    fontSize: 14,
+  },
+});
+
+const StyledLabel = styled("span")({
+  fontSize: "0.7rem",
+});
+
+const SignIn = () => {
+  const theme = useTheme();
+  const lineColor = theme.palette.primary.main;
+  const [username, setUsername] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```

Comment:
would be better to take state in a object, and give textbox name which exactly matches to keys of the state\
`
interfact LoginStateDto {
userName: string,
password: string
}
const defaultLoginState : LoginStateDto = {
userName: "",
password: ""
}
const [logInState, setLogInState] = useState<LoginStateDto>(defaultLoginState)`